src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -319,6 +319,12 @@ reg_class int_rdi_reg(RDI);
 //----------SOURCE BLOCK-------------------------------------------------------
 // This is a block of C++ code which provides values, functions, and
 // definitions necessary in the rest of the architecture description
+
+source_hpp %{
+  #include CPU_HEADER(peephole)
```

Comment:
Why don't you simply include `peephole_x86_64.hpp` here?

Reply:
Done